packages/esm-patient-list-app/src/patient-list-detail/patient-list-detail.component.tsx

```diff
@@ -38,19 +39,23 @@ interface PatientListDetailProps {
   patientListUuid: string;
 }
 
-const PatientListDetailComponent: React.FC<RouteComponentProps<PatientListDetailProps>> = ({ match }) => {
+const PatientListDetailComponent: React.FC<RouteComponentProps<PatientListDetailProps>> = () => {
   const patientListUuid = getPatientListUuidFromUrl();
   const { t } = useTranslation();
   const [currentPage, setPageCount] = useState(1);
   const [currentPageSize, setCurrentPageSize] = useState(10);
   const [searchString, setSearchString] = useState('');
-  const { data: patientListDetails } = usePatientListDetails(patientListUuid);
+  const { data: patientListDetails, mutate: patientListDetailsMutate } = usePatientListDetails(patientListUuid);
   const { data: patientListMembers } = usePatientListMembers(
     patientListUuid,
     searchString,
     (currentPage - 1) * currentPageSize,
     currentPageSize,
   );
+  const [editPatientListDetail, showOverlay] = useState(false);
+  const handleSuccessEdit = useCallback(() => {
+    patientListDetailsMutate();
+  }, [patientListDetailsMutate]);
```

Comment:
That seems redundant - I'd just remove the entire thing and call/pass `patientListDetailsMutate` directly.

Reply:
But isn't it a good option to have callbacks in `useCallback`. SInce it's also a callback, it should be wrapped inside the same.

Reply:
I'm open to be corrected, but generally `useCallback` should *not* be used to wrap every callback. Brandon linked a good blog post about this some time ago: https://kentcdodds.com/blog/usememo-and-usecallback. I recommend reading it, it will only take a few minutes, but is (in my opinion) really helpful for getting a feel for when `useCallback` is required.

In this situation here `useCallback` will only cause overhead. The wrapped functions should already be referentially stable (i.e. the reference doesn't change between renders. SWR ensures that.). -> There's no reason to again wrap it with `useCallback`.

Reply:
Thank you @manuelroemer for the help and the link to the blog. I have made the necessary changes.